src/_cffi_src/build_openssl.py

```diff
@@ -38,8 +38,15 @@ def _osx_libraries(build_static):
 
 
 _PRE_INCLUDE = """
+#include <openssl/opensslv.h>
+/*
+    LibreSSL removed e_os2.h from the public headers so we'll only include it
+    if we're using vanilla OpenSSL.
+*/
```

Comment:
Ugh, help me roll this back: When is `OPENSSL_SYS_WINDOWS` set but `_WIN32` isn't it?